### PR TITLE
cpu/cc26x0: rename to cc26x0_cc13x0

### PR DESCRIPTION
**Pull request by jeandudey**
### Contribution description

Renames the cc13x0 CPU (as I did before with cc26x2 cpu) in order to add support to the cc13x0 family of micro controllers.

### Testing procedure

cc26x0 boards should still compile and work:

- `make -C examples/hello-world BOARD=cc2650-launchpad`
- `make -C examples/hello-world BOARD=cc2650stk`

### Issues/PRs references

### CODE REVIEW

**Comment by benpicco** [comment]
CI spotted some issues.

**Comment by benpicco** [comment]
You want to also check `tests/kconfig_features` for the affected boards, Murdock found an issue there

File: boards/cc2650-launchpad/Makefile.features [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CPU = cc26x0
+CPU = cc26x0_cc13x0
```
Comment (benpicco) [inline boards/cc2650-launchpad/Makefile.features:1]: Kconfig needs an update too

File: boards/cc2650stk/Makefile.features [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CPU = cc26x0
+CPU = cc26x0_cc13x0
```
Comment (benpicco) [inline boards/cc2650stk/Makefile.features:1]: here too

**Comment by benpicco** [comment]
Please squash!

**Review by benpicco — APPROVED** [review]
looks good to me.

**Comment by jeandudey** [comment]
Thanks for the review :-)

**Pull request merged**